drivers/ethernet/CMakeLists.txt

```diff
@@ -6,3 +6,7 @@ zephyr_sources_ifdef(CONFIG_ETH_DW		eth_dw.c)
 zephyr_sources_ifdef(CONFIG_ETH_ENC28J60	eth_enc28j60.c)
 zephyr_sources_ifdef(CONFIG_ETH_MCUX		eth_mcux.c)
 zephyr_sources_ifdef(CONFIG_ETH_STM32_HAL	eth_stm32_hal.c)
+zephyr_sources_ifdef(CONFIG_ETH_NATIVE_POSIX
```

Comment:
Would you consider adding a few lines for this driver in http://docs.zephyrproject.org/boards/posix/native_posix/doc/board.html#peripherals ?

Reply:
Sure, np